testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -386,15 +387,26 @@ impl BlockMinerThread {
                             return Err(e);
                         }
                         _ => {
-                            error!("Error while gathering signatures: {e:?}. Will try mining again.";
+                            // Sleep for a bit to allow signers to catch up
+                            let pause_ms = if last_block_rejected {
+                                self.config.miner.subsequent_rejection_pause_ms
+                            } else {
+                                self.config.miner.first_rejection_pause_ms
+                            };
+                            thread::sleep(Duration::from_millis(pause_ms));
+                            last_block_rejected = true;
+
+                            error!("Error while gathering signatures: {e:?}. Will try mining again in {pause_ms}.";
```

Comment:
should move this error log to above your sleep i think.

Reply:
Yup. Moved in 6e0eca60969f0029f67dc259790dbbb30391ad22.